integration/conftest.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""pytest configuration and global fixtures for integration tests."""
+import json
+
+import pytest
+
+import firebase_admin
+from firebase_admin import credentials
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--cert', action='store', help='Service account certificate file for integration tests.')
+
+def _get_cert_path(request):
+    cert = request.config.getoption('--cert')
+    if cert:
+        return cert
+    raise ValueError('Service account certificate not specified. Make sure to specify the '
+                     '"--cert" command-line option.')
+
+@pytest.fixture(autouse=True, scope='session')
+def default_app(request):
+    """Initializes the default Firebase App instance used for all integration tests.
+
+    This fixture is attached to the session scope, which ensures that it runs only once during
+    a test session. It is also marked as autouse, and therefore runs automatically without
+    test cases having to call it explicitly.
+    """
+    cert_path = _get_cert_path(request)
+    with open(cert_path) as cert:
+        project_id = json.load(cert).get('project_id')
```

Comment:
It seems the `project_id` field in the service account key file we download from Firebase console is same as the realtime database name of the project. We have been using this convention in other 2 Admin SDKs too for testing. I'll try to get it confirmed from the database server team.